cpp/include/Ice/Connection.h

```diff
@@ -431,6 +431,68 @@ namespace Ice
         {
         }
     };
+
+    /**
+     * Provides access to the connection details of an IAP connection
+     * \headerfile Ice/Ice.h
+     */
+    class IAPConnectionInfo final : public ConnectionInfo
```

Comment:
I didn't update the Swift code yet even though this renaming makes an update necessary.

I am curious to see if the Swift CI will fail.

Reply:
Seems we are not building for iOS, there is some commented out code in the ci workflow:

https://github.com/zeroc-ice/ice/blob/8c76edbd3441d83394995c7a08761ce707966ed5/.github/workflows/ci.yml#L55-L62